### PR TITLE
Fusetools 2493 avoid to have space value for enum choice

### PR DESCRIPTION
**Pull request by apupier**
review and merge this one first for simplicity reviewing this one https://github.com/jbosstools/jbosstools-fuse/pull/1058

### CODE REVIEW

File: core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelComponentUtils.java [hunk 1]
```diff
@@ -183,10 +183,10 @@ public static boolean isUnsupportedProperty(Parameter p) {
 		return isMapProperty(p) || p.getJavaType().toLowerCase().startsWith("java.util.date");
 	}
 
-	public static String[] getChoices(Parameter p) {
-		String[] choices = p.getChoice().split(",");
+	public static String[] getChoicesWithExtraEmptyEntry(Parameter p) {
+		String[] choices = p.getChoice();
 		List<String> res = new ArrayList<>();
-		res.add(" "); // empty entry
+		res.add(""); // empty entry
```
Comment (apupier) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelComponentUtils.java]: this is the only change

**Comment by apupier** [comment]
I'm not sure that a unit test is relevant here. A UI one seems better, agreed?

**Comment by lhein** [comment]
I agree. UI tests make more sense here.

**Comment by apupier** [comment]
I created https://issues.jboss.org/browse/FUSETOOLS-2500 for the UI test

**Pull request merged**